### ISSUE TITLE
Button progress does not work on flat or outlined styles

### ISSUE THREAD

**Issue by ymhr** [quasarframework/quasar#6708]
**Describe the bug**
The progress bar (combination of `loading` and `percentage` props) does not display on the `outlined` or `flat` style of the button. I can see from the inspector that the element is there, but it does not show.

**Codepen/jsFiddle/Codesandbox (required)**
https://codepen.io/ymhr/pen/QWbYaXb?editable=true&editors=101

**Expected behavior**
The progress bar should show no matter what style is active.

**Comment by pdanpdan** [comment]
Good catch. Until it's fixed you can use something like this:

```styl
.q-btn--flat, .q-btn--outline
  .q-btn__progress-indicator
    opacity: 0.1
    background-image: linear-gradient(to right, currentColor, currentColor), linear-gradient(to right, rgba(255,255,255,.25), rgba(255,255,255,.25))
    
    &--dark
      background-image: linear-gradient(to right, currentColor, currentColor), linear-gradient(to right, rgba(0,0,0,.2), rgba(0,0,0,.2))
```

https://codepen.io/pdanpdan/pen/eYNxVey

**Comment by rstoenescu** [comment]
Fix will be available in "quasar" v1.9.13.
